### PR TITLE
Add links directing to integrations

### PR DESCRIPTION
**Pull request by WanYixian**
## Description

As suggested in [slack](https://risingwave-labs.slack.com/archives/C034XCYJPSN/p1732083200854659), update the sidebar. Click [preview](https://risingwavelabs-wyx-add-direct-link.mintlify.app/ingestion/overview) to check if this is the ideal result, let me know if any comments, thanks!

1. Under Guides tab, add 3 auto-direct pages pointing to Integrations tab:
image
image
image

2. Under Integrations tab, link back to Guides tab.

But these are landing pages with card components matching the left-side toc precisely. Adding 3 extra pages like above may mess it up, so add one-line description for each to link back to Guides.
image

image
image

### CODE REVIEW

File: mint.json [hunk 1]
```diff
@@ -422,6 +423,7 @@
           "group": "Process data",
           "pages": [
             "processing/overview",
+            "processing/sql-references",
```
Comment (xxchan) [inline mint.json]: put it under "sql"?

**Review by xxchan — COMMENTED** [review]
What about we also add link back to source/sink docs in "integration"'s sidebar?

File: delivery/guides-on-supported-sinks.mdx [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+title: Guides on supported sinks
```
Comment (xxchan) [inline delivery/guides-on-supported-sinks.mdx]: what about 
```suggestion
title: Supported sink connectors
```

**Comment by WanYixian** [comment]
> What about we also add link back to source/sink docs in "integration"'s sidebar?

Updated, thanks!

BTW, This [overview](https://risingwavelabs-wyx-add-direct-link.mintlify.app/integrations/sources/overview) serves as the landing page for its subsections, so instead of adding one page paralleled with it to link back to [source](https://risingwavelabs-wyx-add-direct-link.mintlify.app/ingestion/overview), I included a one-line description within it.
image

File: mint.json (hunk 1 above)
Comment (WanYixian) [reply mint.json]: > put it under "sql"?

Updated, but putting the auto-direct page like this as the first item under a section doesn't work well, so I moved it to the last one under SQL section
image

**Comment by WanYixian** [comment]
Please help take a look at your earliest convenience, thank you 😃

**Review by st1page — APPROVED** [review]

**Pull request merged**